app/src/main/java/org/oppia/android/app/help/HelpFragmentPresenter.kt

```diff
@@ -12,6 +12,8 @@ import org.oppia.android.app.viewmodel.ViewModelProvider
 import org.oppia.android.databinding.HelpFragmentBinding
 import org.oppia.android.databinding.HelpItemBinding
 import javax.inject.Inject
+import kotlinx.android.synthetic.main.help_item.*
+import kotlinx.android.synthetic.main.help_item.view.*
```

Comment:
We shouldn't use synthetic generated code anymore--it's been deprecated in favor of databinding or Jetpack Compose (we use the former currently in Oppia Android).